client/components/course/Outline/index.vue

```diff
@@ -64,7 +64,13 @@ export default {
         .sort((x, y) => x.position - y.position);
     }
   },
-  methods: mapMutations(['toggleActivities'], 'course'),
+  methods: {
+    ...mapActions({ getActivities: 'fetch' }, 'activities'),
+    ...mapMutations(['toggleActivities'], 'course')
+  },
+  created() {
+    this.getActivities();
```

Comment:
If you did fetch again here because you wanted updated state after you restored something on history page then it makes more sense to do it [there where you restored it](https://github.com/ExtensionEngine/tailor/blob/7a8de47ed8cc060b0def150bfa05bef5d7b4e927/client/components/course/Revisions/RevisionItem.vue#L72-L76). Also use ```reset``` instead of ```fetch```.